src/glwrapper.rs

```diff
@@ -847,6 +847,8 @@ impl GLContextManager
             }))
         };
 
+        manager.set_enable_scissor(true);
```

Comment:
I think it would be best to only enable this the first time the user calls `set_clip` -- this means that users who don't use clipping don't have to experience the performance impact.

Reply:
Do you want to add some field flag to set that would mean that we already have enabled scissor? Or how can we check that we haven't already enabled it?

Reply:
Yeah a flag inside `GLContextManagerState` would make sense, similar to the `active_texture` field used in `bind_texture()`.

We could also disable scissoring at the start of each frame, so the values from the previous frame aren't left over.